viewing_party/main.py

```diff
@@ -0,0 +1,136 @@
+def create_movie(title, genre, rating):
+    movies_dict = {}
+    if title and genre and rating:
+        movies_dict["title"] = title
+        movies_dict["genre"] = genre
+        movies_dict["rating"] = rating
```

Comment:
Nice! Another way to write this code is to create the dictionary in the if statement so

```
if title and genre and rating:
    movies_dict = {
           "title": title,
           "genre": genre,
           "rating": rating
    }
    
    return movies_dict
```